landlab/io/netcdf/write.py

```diff
@@ -322,11 +322,11 @@ def _add_cell_spatial_variables(root, grid, **kwds):
 
```

Comment:
beyond the scope of this PR... but should we change the write netcdf to be based on xarray now?

Reply:
Yes, definitely! I could do a separate pull request for it.

Reply:
I'll make an issue for this to keep track.

Reply:
See Issue #1014